data/scripts/valadoc.js

```diff
@@ -6,9 +6,12 @@ const config = {
 
 const curpkg = window.location.pathname.split('/')[1]
 
+// cache of the content of tooltips stored by their url
+var tooltipCache = {}
+
 // This object will contain the html elements of the interface
 const html = {
-  tooltips: []
+  tooltipEl: initTooltip(),
```

Comment:
Not sure about the trailing comma.

Reply:
I prefer it for cleaner diffs in the future but I'm happy to change it to whatever you prefer.